clients/nginxservice.go

```diff
@@ -73,6 +87,11 @@ func NewNginxService() *corev1.Service {
 			},
 		},
 	}
+
+	ip := svcObj.Spec.LoadBalancerIP
```

Comment:
LoadBalancerIP is set by k8s after deploying it, so this value at this time should always be "", correct?

Reply:
Yes, that is true. This isn't currently being used, I'll delete everything related to this struct for now.